common/cpp/types.h

```diff
@@ -27,16 +27,6 @@
 #include <vector>
 #include <list>
 
-/** Avalon worker enclave Type (singleton, KME, WPE) */
-enum EnclaveType {
-    // SINGLETON_ENCLAVE does both Key management and workload processing
```

Comment:
types.h would be right place to keep enclaveType enum. Any reason you moved to tcf_error.h?

Reply:
I did move the enclave type enum to tcf_error.h because there are other enums in place. Also, types.h exposes lot of APIs like ByteArrayToString, ByteArrayToStringArray along with some typedefs. This was resulting in undefined symbol errors during python module load time. Hence, i moved it to tcf_error.h